integrations/transportation/tfl.js

```diff
@@ -10,11 +10,13 @@ async function request(url, opts) {
   try {
     const response = await fetch(url, opts);
     if (!response.ok) {
-      throw new Error(response);
+      // This will be caught by the catch block and recaught
+      throw new Error(response.statusText);
```

Comment:
here you'd want to throw an `HTTPError` and have the second argument be the http status code

Reply:
I've simplified it and made it more like the other integrations.